pkg/surveyext/editor_manual.go

```diff
@@ -15,8 +15,8 @@ type showable interface {
 	Show()
 }
 
-func Edit(editorCommand, fn, initialValue string, stdin io.Reader, stdout io.Writer, stderr io.Writer, cursor showable) (string, error) {
```

Comment:
Can the `showable` interface now be eliminated too?

Reply:
Looks like it is still used below in the `edit` function which is used in `GhEditor.prompt`.